packages/rolldown/package.json

```diff
@@ -63,11 +63,10 @@
   },
   "devDependencies": {
     "@napi-rs/cli": "^3.0.0-alpha.43",
-    "@types/node": "^20.11.25",
-    "npm-run-all2": "^6.1.2",
-    "prettier": "^3.2.5",
+    "colorette": "^2.0.20",
```

Comment:
There are used in tests, but weren't listed. Because pnpm scopes them, they're now required.